django-check/action.yaml

```diff
@@ -50,13 +56,7 @@ runs:
         shell: bash
         run: |-
           cd ${{ inputs.path }}
-          pipenv run black --check .
+          pipenv run black --check . || pipenv run black --diff .
         if: ${{ inputs.black }}
-      - name: Test (run in parallel)
-        shell: bash
-        run: |-
-          cd ${{ inputs.path }}
-          pipenv run coverage run --concurrency=multiprocessing manage.py test --settings=${{ inputs.projectName }}.settings.ci --parallel
-          pipenv run coverage combine
-  # container:
-  #     image: python:${{ inputs.pythonVersion }}
+  container:
```

Comment:
Any reason we want to bring this back?

Reply:
Do all products use the same python version? If I remember correctly, that's not true, which is the rationale for bringing this back.

Reply:
I'm curious why we removed it, though

Reply:
Let's add it back temporarily and monitor closely.